x-pack/plugins/lens/public/editor_frame_service/embeddable/embeddable.tsx

```diff
@@ -54,6 +56,7 @@ export type LensByValueInput = {
 export type LensByReferenceInput = SavedObjectEmbeddableInput & EmbeddableInput;
 export type LensEmbeddableInput = (LensByValueInput | LensByReferenceInput) & {
   palette?: PaletteOutput;
+  activeData?: TableInspectorAdapter;
```

Comment:
Is this a leftover? Seems like activeData is never actually passed in as embeddable input (and I don't see a good reason for that).